test/integration/scheduler_perf/util.go

```diff
@@ -17,90 +17,35 @@ limitations under the License.
 package benchmark
 
 import (
-	"net/http"
-	"net/http/httptest"
-
-	"github.com/golang/glog"
 	"k8s.io/api/core/v1"
-	"k8s.io/client-go/informers"
 	clientset "k8s.io/client-go/kubernetes"
-	clientv1core "k8s.io/client-go/kubernetes/typed/core/v1"
 	restclient "k8s.io/client-go/rest"
-	"k8s.io/client-go/tools/record"
-	"k8s.io/kubernetes/pkg/api/legacyscheme"
 	"k8s.io/kubernetes/pkg/api/testapi"
 	"k8s.io/kubernetes/pkg/scheduler"
 	_ "k8s.io/kubernetes/pkg/scheduler/algorithmprovider"
-	"k8s.io/kubernetes/pkg/scheduler/factory"
-	"k8s.io/kubernetes/test/integration/framework"
+	"k8s.io/kubernetes/test/integration/util"
 )
 
-const enableEquivalenceCache = true
-
 // mustSetupScheduler starts the following components:
 // - k8s api server (a.k.a. master)
 // - scheduler
 // It returns scheduler config factory and destroyFunc which should be used to
 // remove resources after finished.
 // Notes on rate limiter:
 //   - client rate limit is set to 5000.
-func mustSetupScheduler() (schedulerConfigurator scheduler.Configurator, destroyFunc func()) {
-
-	h := &framework.MasterHolder{Initialized: make(chan struct{})}
-	s := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
-		<-h.Initialized
-		h.M.GenericAPIServer.Handler.ServeHTTP(w, req)
-	}))
-
-	framework.RunAMasterUsingServer(framework.NewIntegrationTestMasterConfig(), s, h)
-
+func mustSetupScheduler() (scheduler.Configurator, util.ShutdownFunc) {
+	masterURL, masterShutdown := util.StartApiServer()
```

Comment:
s/Server/server/

Reply:
you mean `startApiserver()`?

Reply:
I meant StartApiserver.. The 's' in apiserver I think is kept small based on convention from other places in the code.